mllib/src/main/scala/org/apache/spark/ml/recommendation/ALS.scala

```diff
@@ -200,8 +240,42 @@ class ALSModel private[ml] (
   @Since("1.3.0")
   def setPredictionCol(value: String): this.type = set(predictionCol, value)
 
+  /** @group setParam */
   @Since("2.0.0")
+  def setLabelCol(value: String): this.type = set(labelCol, value)
+
+  /** @group setParam */
+  @Since("2.0.0")
+  def setK(value: Int): this.type = set(k, value)
+
+  /** @group setParam */
+  @Since("2.0.0")
+  def setRecommendFor(value: String): this.type = set(recommendFor, value)
```

Comment:
Would it make sense to set `recommendFor` and `k` together?  like this
```
def setRecommendFor(forval: String, kval: Int)
```
That way you would be guaranteed they are both set if using recommendFor